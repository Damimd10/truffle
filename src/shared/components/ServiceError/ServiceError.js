import React from 'react';
import styled from 'styled-components';

const ErrorContainer = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100vh;
  font-size: 25px;
  color: #591919;
  font-weight: bold;
  text-transform: uppercase;
`;

const ServiceError = ({ message }) => <ErrorContainer>{message}</ErrorContainer>;

export default ServiceError;
