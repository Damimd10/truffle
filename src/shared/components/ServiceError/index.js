import ServiceError from './ServiceError';

export default ServiceError;
