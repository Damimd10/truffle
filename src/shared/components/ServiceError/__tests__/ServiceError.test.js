import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import ServiceError from '../ServiceError';

describe('Service Error Component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ServiceError message="Some message" />);
  });

  it('should display a render of styled component container', () =>
    expect(toJson(wrapper)).toMatchSnapshot());
});
