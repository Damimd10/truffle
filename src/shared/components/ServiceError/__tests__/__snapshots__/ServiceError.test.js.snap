// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Service Error Component should display a render of styled component container 1`] = `
<styled.div>
  Some message
</styled.div>
`;
